qa/tasks/cephfs/caps_helper.py

```diff
@@ -13,10 +13,9 @@ def run_mon_cap_tests(self, moncap, keyring):
         fsls = self.run_cluster_cmd(f'fs ls --id {self.client_id} -k '
                                     f'{keyring_path}')
 
-        # we need to check only for default FS when fsname clause is absent
-        # in MON/MDS caps
-        if 'fsname' not in moncap:
-            self.assertIn(self.fs.name, fsls)
+        if 'fsname=' not in moncap:
+            fsls_admin = self.fs.mon_manager.raw_cluster_cmd(args='fs ls')
```

Comment:
Any reason for not using self.run_cluster_cmd as done in the line above?

Reply:
I was just trying to be closer to the PR to which this commit originally belonged to.